scss/datetime/_layout.scss

```diff
@@ -135,6 +135,7 @@
     }
 
     $time-list-wrapper-padding: $padding-y * 8;
+    $time-list-wrapper-focused-border-width: 2px;
```

Comment:
No need for `-wrapper` in the variable names. I'd even go further and shorten the latter to `$time-list-focus-size`, which will also hide the implementation detail that it is achieved with `border` (we can change to `box-shadow` and the name will still make sense).